pumpkin/src/client/container.rs

```diff
@@ -82,4 +85,20 @@ impl Player {
             &item.into(),
         ))
     }
+
+    /// The official Minecraft client is weird, and will always just close *any* window that is opened when this gets sent
```

Comment:
Why that is weird ?

Reply:
I should probably change the wording from weird, but it's mainly just a note so that people won't try and depend on behaviour that only a specific window will be closed, when which window_id you send doesn't actually matter to the client

Reply:
What I think I meant to comment on as weird is that the packet still requires an ID, but it never gets used for anything.